### ISSUE TITLE
Point lines

### ISSUE THREAD

**Issue by clauswilke** [thomasp85/ggforce#198]
Would you be willing to accept a PR that adds geoms drawing point lines (horizontal or vertical lines that behave like points)? See here: https://wilkelab.org/ungeviz/articles/misc-geoms-stats.html#horizontal-and-vertical-plines

I'm happy to use different names for the geoms if you don't like my current naming.

**Comment by thomasp85** [comment]
Absolutely. PR away

**Comment by thomasp85** [comment]
and this?

**Comment by clauswilke** [comment]
Yeah. I still think it's a good idea but don't know if I'll ever get around to it. If I do I'll just prepare a PR.
